### DISCUSSION TITLE
Is it wise to use NodeJS' builti-in Single Executable Application for production in v22.x?

### DISCUSSION THREAD

**Discussion by ak2766** [nodejs/single-executable#113] [General]
I'm currently limited to using NodeJS v20.x since `pkg` doesn't have capability to bundle into v22.x.  As such, I'm contemplating switching to SEA but my search for whether this is ready for production use hasn't yielded any results to sway me to confidently use it in production.

I have posed this question to ChatGPT it stated the feature is still experimental but I can't trust it full due to hallucination concerns.

Is there a web page somewhere where the maturity of this feature is documented?

**Comment by GabenGar** [comment]
It's literally in the docs (which is a first result for `node sea`): 
https://nodejs.org/api/single-executable-applications.html

**Comment by ak2766** [comment]
Are you saying we are now supposed to infer stuff from the use of an `--experimental-sea-config` flag?  That's just absurd!  Without it clearly being stated, it's going to be open to misinterpretation!

I guess it is what it is.

Thanks for responding anyway.
